package.json

```diff
@@ -124,6 +124,7 @@
         "@babel/types": "^7.24.7",
         "@box/blueprint-web": "^7.19.8",
         "@box/blueprint-web-assets": "^4.21.0",
+        "@box/box-ai-content-answers": "^0.49.1",
```

Comment:
had to add at least one of the shared-features in here for it to account for i18n bundles